docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -1111,6 +1111,24 @@ UDP
   UDP
     Not implemented ("-").
 
+%DOWNSTREAM_PEER_CHAIN_FINGERPRINTS_256%
```

Comment:
```suggestion
%DOWNSTREAM_PEER_CHAIN_FINGERPRINTS_SHA256%
```

Reply:
I considered the more detailed name, but opted to match the leaf cert variants; do you think it's worth having two different naming schemes?

Reply:
I hadn't noticed that; sticking with existing convention seems fine